Notebooks/Bruxism_detection.ipynb

```diff
@@ -29,7 +29,8 @@
      "name": "stdout",
```

Comment:
<p>Important:</p><p>Is 5e-4 the new emg THR you advise for the is_good algorithm?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
@RobinGuillard actually, 5e-4 will remove all HUGE artefacts and I wanted so until we see that it is a problem. 
The issue being mixing artifacts and bursts in the episode because the is_good is too sensible.